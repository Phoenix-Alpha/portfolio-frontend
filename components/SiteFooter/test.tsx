import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SiteFooter } from './index'

describe('<SiteFooter />', () => {
  it('should renders correct <SiteFooter />', async () => {
    const wrapper = shallow(<SiteFooter />)

    await new Promise((resolve) => setTimeout(resolve, 0))

    wrapper.update()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
