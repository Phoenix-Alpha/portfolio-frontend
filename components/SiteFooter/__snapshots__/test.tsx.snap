// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteFooter /> should renders correct <SiteFooter /> 1`] = `
<footer
  className="fixed bottom-0 w-full backdrop-blur-md"
>
  <Container
    className="flex flex-col px-5 lg:px-16 w-full lg:flex-no-wrap"
  >
    <Row>
      <Divider
        className="!my-0 bg-[#333333]"
      />
    </Row>
    <Row
      className="flex-row justify-between my-4"
    >
      <span
        className="font-body text-white"
      >
        Â© Betafoenix 2022.
      </span>
      <SocialLinks
        className="flex flex-row space-x-4 lg:space-x-10"
      />
    </Row>
  </Container>
</footer>
`;
